Documentation/Development/CodingStandards.md

```diff
@@ -0,0 +1,39 @@
+# WheelMUD Coding Standards
+
+## StyleCop
+We are using StyleCop to help us keep a handle on coding standards and consistency throughout the code-base.
```

Comment:
I'm not a fan of style-cop myself per our other talks I'm using Jetbrains REesharper.. is this an issue?